utils/src/main/scala/ai/h2o/sparkling/utils/SparkSessionUtils.scala

```diff
@@ -0,0 +1,30 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package ai.h2o.sparkling.utils
+
+import org.apache.spark.sql.SparkSession
+
+/**
+ * Internal utilities methods for Spark Session
+ */
+object SparkSessionUtils {
```

Comment:
We already have the `SparkSessionUtils` object in the `org.apache.spark` namespace. Should we merge these two objects together?

Reply:
Yup, I have thought about that, but let's do it in a separate PR. Thanks for checking!

Reply:
https://0xdata.atlassian.net/browse/SW-1982